package.json

```diff
@@ -33,7 +33,11 @@
     "@fortawesome/react-fontawesome": "^0.1.8",
     "@gnosis.pm/dex-js": "^0.4.1",
     "@hapi/joi": "^17.1.1",
+    "@hookform/error-message": "^0.0.3",
+    "@hookform/resolvers": "^0.0.6",
```

Comment:
Don't see this used anywhere. A required peer dependency?

Reply:
not required - think i _was_ using it elsewhere. good catch